charts/datafold/charts/dfshell/templates/deployment.yaml

```diff
@@ -41,6 +41,60 @@ spec:
                 name: {{ include "datafold.configmap" . }}
           env:
             {{ include "datafold.env" . | nindent 12 }}
+            - name: DATAFOLD_FRESHPAINT_FRONTEND_TOKEN
```

Comment:
Do we really want all of these secrets to be propagated to all containers?  As part of this deployment, I wanted to limit that to the containers that need it only.